pkg/controller/nexus/resource/networking/manager.go

```diff
@@ -204,9 +178,5 @@ func ingressEqual(deployed resource.KubernetesResource, requested resource.Kuber
 	pairs = append(pairs, [2]interface{}{ingress1.Namespace, ingress2.Namespace})
 	pairs = append(pairs, [2]interface{}{ingress1.Spec, ingress2.Spec})
 
-	equal := compare.EqualPairs(pairs)
-	if !equal {
-		log.Info("Resources are not equal", "deployed", deployed, "requested", requested)
```

Comment:
This log is pretty useful. Are you sure you want to remove it?

Reply:
Ugh, not sure how that got in there. Thanks for pointing it out, definitely want to keep it :-)